web/src/engine/js-processor/src/defaultRules.ts

```diff
@@ -224,3 +217,9 @@ export default class DefaultRules {
     return null;
   }
 }
+
+/**
+ * A simple instance of the standard 'default rules' for keystroke processing from the
+ * DefaultRules base class.
+ */
+BASE_DEFAULT_RULES.instance = new DefaultRules();
```

Comment:
I see where this `BASE_DEFAULT_RULES` plugs in now. I am uncomfortable with this; can you explain the rationale?